airflow/cli/commands/db_command.py

```diff
@@ -61,8 +61,8 @@ def shell(args):
                 [client]
                 host     = {url.host}
                 user     = {url.username}
```

Comment:
```suggestion
                user     = {url.username or ""}
```
Is Username  optionally?